tests/py/test-jupyter.cpp

```diff
@@ -21,4 +21,4 @@ class JupyterTests : public PythonTests {
 #define TEST_JUPYTER(notebook) EXPECT_NO_THROW(\
         PYTHON_TEST_CALL0("test_notebook", python::object(URI::resolve(notebook))));
 
-TEST_F(JupyterTests, python_kb) { TEST_JUPYTER("jupyter/intro.ipynb"); }
+TEST_F(JupyterTests, python_kb) { TEST_JUPYTER("tutorials/intro.ipynb"); }
```

Comment:
do we need this test at all still? if it is redundant with the python testing, I think we could drop the JupyterTests class